lib/src/app.dart

```diff
@@ -140,8 +147,8 @@ class App {
   /// Starts the HTTP server listening on the specified port
   ///
   /// All Request and Response objects will be wrapped and handled by the Router
-  listen(int port, [Function(int) cb]) async {
-    this._server = await HttpServer.bind(InternetAddress.loopbackIPv4, port);
+  listen(InternetAddress address, int port, [Function(int) cb]) async {
```

Comment:
Thank you for the contribution, @jeffmikels! This is a good change.

What do you think about changing these all into named parameters since this is going to break for people upgrading anyways. I think `named` parameters provide a little nicer interface. We could make the address optional still and make it easier to add future configuration without breaking the interface completely.

I'm thinking something like this. If you'd like to make that change, go for it or I can always make it after this is merged.

```dart
listen({
  InternetAddress address = InternetAddress.loopbackIPv4,
  @required int port,
  Function(int) cb
}) async {
  // ...
}
```